bull-common/src/test/java/com/hotels/transformer/utils/ReflectionUtilsTest.java

```diff
@@ -153,6 +154,21 @@ public void testGetFieldValueWithAGivenFieldWorksAsExpected() throws Exception {
         };
     }
 
+    /**
+     * Tests that the method {@code getFieldValue} catches InvalidBeanException.
```

Comment:
I think this title is wrong, it should say: 

>  catches RuntimeException,

moreover, this is a minor thing, could you please replace `{@code getFieldValue}` with `{@link ReflectionUtils#getFieldValue(Object, String, Class) getFieldValue}`